cmd/cloud.go

```diff
@@ -64,13 +63,13 @@ func (c *cmdCloud) preRun(cmd *cobra.Command, args []string) error {
 // TODO: split apart some more
 //nolint:funlen,gocognit,cyclop
 func (c *cmdCloud) run(cmd *cobra.Command, args []string) error {
-	printBanner(c.gs)
+	c.gs.console.Printf("\n%s\n\n", c.gs.console.Banner())
```

Comment:
I overlooked it, that's why :sweat_smile: Nice catch!

Fixed in 22982999a.

I'm against encapsulating the check, since it's more readable to have the logic that decides whether to do something (print the banner in this case) outside of any functions, unless the check is overly verbose or is used in many places. Previously, `printBanner()` did that check internally, but now it's clearer that the banner is only printed if `quiet` wasn't enabled.

In general, I'm not a fan of the pattern of calling a function that returns right away depending on some check, especially if that check can be done by the caller. Functions should do what it says on the tin; nothing more and nothing less. :slightly_smiling_face:

We could always have another `maybePrintBanner()` or `printBannerUnlessQuiet()` function, but I think that's a bit silly considering it only saves us 2 lines, and it's called from 2 places.

Notice that I also didn't encapsulate the actual printing of the banner, since calling `console.Printf()` directly is also clearer.

Reply:
>I'm against encapsulating the check, since it's more readable to have the logic that decides whether to do something 

I see where you are coming from, but generally can't agree :sweat_smile: 

Looking at all usages of the Banner right now, we see everywhere the if statement, and considering it, that likely there will be no place that won't use the same logic (`maybePrintBanner`) having these `if` are unnecessary details. Moreover, it can lead to errors because the maintainer/contributor can easily forget to put in the additional check before placing the banner.

Reply:
I prefer favoring clarity, even at the expense of verbosity or duplication, so I don't see the external check as an unnecessary detail. The only reason I forgot to add it in this case is because I removed the `printBanner()` function, and forgot to update this usage of it. Since it only happens in 2 places, I doubt this would happen again.

But since you feel strongly about it, I added a `maybePrintBanner` function to `cmd` in 6ca990ae3. I also noticed the same issue with printing the `cloud` progress bar, so also added `maybePrintBar`. Let me know if this addresses your concern.

I tested the behavior of `k6 cloud`, with and without `--quiet`, and it now matches the behavior on `master-next`.